The -dark variants of Themes don't support GTK4 yet. As such, strip the -dark
suffix from themes. Instead use theme_name:dark if you do want to use a dark mode.
diff --git a/src/hdy-main.c b/src/hdy-main.c
index 24f83b9..39cdc72 100644
--- a/src/hdy-main.c
+++ b/src/hdy-main.c
@@ -52,6 +52,12 @@ hdy_themes_get_theme_name (gboolean *prefer_dark_theme)
                   "gtk-application-prefer-dark-theme", prefer_dark_theme,
                   NULL);
 
+    if (g_str_has_suffix(theme_name, "-dark")) {
+      gchar** arr = g_strsplit(theme_name, "-dark", 2);
+      theme_name = g_strdup(arr[0]);
+      g_strfreev(arr);
+    }
+
     return theme_name;
   }
 
@@ -63,6 +69,12 @@ hdy_themes_get_theme_name (gboolean *prefer_dark_theme)
     *prefer_dark_theme = g_strcmp0 (p, "dark") == 0;
   }
 
+  if (g_str_has_suffix(theme_name, "-dark")) {
+    gchar** arr = g_strsplit(theme_name, "-dark", 2);
+    theme_name = g_strdup(arr[0]);
+    g_strfreev(arr);
+  }
+
   return theme_name;
 }
 
